blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
```

Comment:
Please follow the markdown templete for blog posts. Document details should go like this:
 `--- <details - Title to categories> --- `